index.dd

```diff
@@ -98,8 +96,14 @@ void main()
 ----
 )
 
+) $(COMMENT your-code-here)
+))) $(COMMENT intro, div, div)
+
+$(COMMENT
+Picking the frontpage example after the page has loaded leads to reflows
+and re-rendering. Hence, we pick the example as soon as possible
 )
-)))
+<script src="$(ROOT_DIR)js/example_roulette.js"></script>
```

Comment:
I still would like to add the example selection as well (so the file will grow) and doing Js in Ddoc is really a pain.
Didn't you believe me, when [I showed that using a separate file doesn't matter performance-wise](https://github.com/dlang/dlang.org/pull/1757#issuecomment-310866543)?